frontend/src/services/searchfetcher/APISearchFetcher.ts

```diff
@@ -16,7 +16,6 @@ export class APISearchFetcher extends SearchFetcher {
 
   async fetchOpportunities(): Promise<SearchResponseData> {
     try {
-      await new Promise((resolve) => setTimeout(resolve, 1250));
```

Comment:
Remove the delay (it was just there to see the loader in action)

Reply:
Cool. Maybe just comment it out for now with a TODO to remove it long-term? We're likely gonna iterate on the loader and having this here will be convenient.

Reply:
I see , yea I'll keep it in and commented then